### PR TITLE
handle "rev" parameter

### PR DESCRIPTION
**Pull request by jsteemann**

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/1556/

File: arangod/RestHandler/RestGraphHandler.cpp [hunk 1]
```diff
@@ -289,6 +289,9 @@ void RestGraphHandler::vertexActionRead(
     revision =
         UINT64_MAX;  // an impossible rev, so precondition failed will happen
   }
+  if (revision == 0 || revision == UINT64_MAX) {
+    revision = _request->parsedValue("rev", revision);
+  }
   auto maybeRev = boost::make_optional(revision != 0, revision);
```
Comment (goedderz) [inline arangod/RestHandler/RestGraphHandler.cpp]: @jsteemann You could use `handleRevision()` here as well.

File: arangod/RestHandler/RestGraphHandler.cpp [hunk 2]
```diff
@@ -570,6 +573,9 @@ void RestGraphHandler::edgeActionRead(
...
```
Comment (goedderz) [inline arangod/RestHandler/RestGraphHandler.cpp]: @jsteemann You could use `handleRevision()` here as well.

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/1563/

**Pull request merged**